composer.json

```diff
@@ -35,5 +35,9 @@
         "branch-alias": {
             "dev-master": "1.0.x-dev"
         }
-    }
+    },
+    "bin": [
+        "bin/doctrine-migrations",
+        "bin/doctrine-migrations.php"
```

Comment:
@stof To be honest, I only use the version without the .php. However, I was being consistent with the way this was done in https://github.com/doctrine/doctrine2/blob/master/composer.json

I didn't want to assume that people didn't also want to be able to do
```
php ./vendor/bin/doctrine-migrations.php
```
if they wanted to.